spark/hbase-spark/src/test/java/org/apache/hadoop/hbase/spark/TestJavaHBaseContext.java

```diff
@@ -69,21 +70,29 @@ public class TestJavaHBaseContext implements Serializable {
   public static final HBaseClassTestRule TIMEOUT =
       HBaseClassTestRule.forClass(TestJavaHBaseContext.class);
 
-  private static transient JavaSparkContext JSC;
-  private static HBaseTestingUtility TEST_UTIL;
```

Comment:
Changes in fiele scope at L73, 74 and 75 are not needed as we are not using them in extended class. Will fix in next commit, as part of any other review comments, if received, on the PR.